apps/condo/domains/billing/schema/BillingIntegration.js

```diff
@@ -70,6 +72,14 @@ const BillingIntegration = new GQLListSchema('BillingIntegration', {
             defaultValue: BILLING_INTEGRATION_ORGANIZATION_CONTEXT_IN_PROGRESS_STATUS,
         },
 
+        dataFormat: {
```

Comment:
JSON field is hard to migrate. Maybe we can migrate it to custom field type?

Reply:
I don't think we need to do that. In this particular case the format of this field might change quickly as we add new billing integrations 

Changing JSONField validator is cheaper than changing migrations and writing custom field type or custom schema for this fields seems redundant since it is not business-critical